x-pack/legacy/plugins/reporting/server/browsers/install.ts

```diff
@@ -40,5 +47,7 @@ export async function installBrowser(logger, browserConfig, browserType, install
     await chmod(binaryPath, '755');
   }
 
-  return browser.createDriverFactory(binaryPath, logger, browserConfig, queueTimeout);
+  return {
+    binaryPath,
```

Comment:
This is so the driver factory can instead be created from `create_browser_driver_factory`, and this file is mostly responsible for `install`, so the code should be more readable